tests/manual-test-cases/Group19-VIC-Product/19-5-Configuration-1-VC6.5-ESX6.5-VIC-OVA-Harbor-Secured-SelfSignedCACert.robot

```diff
@@ -0,0 +1,232 @@
+    # Docker pull from harbor using VCH2
+    Log To Console  docker pull from dockerhub VCH2...
+    ${rc}  ${output}=  Run And Return Rc And Output  docker ${VCH2-URL} pull %{HARBOR_IP}/${project}/${image}
+    Log  ${output}
+    Should Be Equal As Integers  ${rc}  0
+
+    # Docker run image
+    Log To Console  docker run VCH2...
+    ${rc}  ${output}=  Run And Return Rc And Output  docker ${VCH2-URL} run --name ${container_name} %{HARBOR_IP}/${project}/${image} /bin/ash -c "dmesg;echo END_OF_THE_TEST" 
+    Log  ${output}
+    Should Be Equal As Integers  ${rc}  0
+    Should Contain  ${output}  END_OF_THE_TEST
+
+    # Docker rm container
+    Log To Console  docker rm VCH2...
+    ${rc}  ${output}=  Run And Return Rc And Output  docker ${VCH2-URL} rm -f ${container_name}
+    Log  ${output}
+    Should Be Equal As Integers  ${rc}  0
+
+    # Docker create
+    Log To Console  docker create VCH2...
+    ${rc}  ${containerID}=  Run And Return Rc And Output  docker ${VCH2-URL} create --name ${container_name} -i %{HARBOR_IP}/${project}/${image} /bin/top
+    Log  ${containerID}
+    Should Be Equal As Integers  ${rc}  0
+
+    # Docker start
+    Log To Console  docker start VCH2...
+    ${rc}  ${output}=  Run And Return Rc And Output  docker ${VCH2-URL} start ${container_name}
+    Log  ${output}
+    Should Be Equal As Integers  ${rc}  0
+
+    # Docker attach 
+    Log To Console  Starting process Docker attach VCH2...
+    Start Process  docker ${VCH2-URL} attach ${container_name} < /tmp/fifo  shell=True  alias=custom
+    Sleep  3
+    Run  echo q > /tmp/fifo
+    ${ret}=  Wait For Process  custom
+    Log  ${ret}
+    Should Be Equal As Integers  ${ret.rc}  0
+    Should Be Empty  ${ret.stdout}
+    Should Be Empty  ${ret.stderr}
+
+    # Docker start  
+    Log To Console  docker start VCH2...
+    ${rc}  ${output}=  Run And Return Rc And Output  docker ${VCH2-URL} start ${container_name}
+    Log  ${output}
+    Should Be Equal As Integers  ${rc}  0
+
+    # Docker stop
+    Log To Console  docker stop VCH2...
+    ${rc}  ${output}=  Run And Return Rc And Output  docker ${VCH2-URL} stop ${container_name}
+    Log  ${output}
+    Should Be Equal As Integers  ${rc}  0
+
+    # Docker remove
+    Log To Console  docker rm VCH2...
+    ${rc}  ${output}=  Run And Return Rc And Output  docker ${VCH2-URL} rm -f ${container_name}
+    Log  ${output}
+    Should Be Equal As Integers  ${rc}  0
+    Wait Until Keyword Succeeds  10x  6s  Check That VM Is Removed  ${container_name}
+    Wait Until Keyword Succeeds  10x  6s  Check That Datastore Is Cleaned  ${container_name}
+
+    # Docker delete image
+    Log To Console  docker rmi VCH2...
+    ${rc}  ${output}=  Run And Return Rc And Output  docker ${VCH2-URL} rmi -f %{HARBOR_IP}/${project}/${image}
+    Log  ${output}
+    Should Be Equal As Integers  ${rc}  0
+    Should Contain  ${output}  Untagged
+
+# Test Pos004 Three Client Machines With One Harbor
```

Comment:
Why isn't this one implemented? Put `Pass Execution . Not implemented because <reason>`